workshops/templates/base_without_navbar.html

```diff
@@ -7,12 +7,12 @@
     <link rel="stylesheet" href="{% static 'css/amy.css' %}" />
     <link rel="stylesheet" href="{% static 'css/bootstrap-datepicker3.min.css' %}" />
     <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />
+    {% block extrastyle %}{% endblock %}
```

Comment:
There's an issue. `django-selectable` has some inclusion template tags that don't use valid XHTML, but our tests rely on XML parsing library so… we have to add `workshops/templates/selectable/jquery-css.html` with this content:
```html
<link rel="stylesheet" href="//ajax.googleapis.com/ajax/libs/jqueryui/{{ version }}/themes/{{ theme }}/jquery-ui.css" type="text/css" />
```

The original file is located here: https://github.com/mlavin/django-selectable/blob/master/selectable/templates/selectable/jquery-css.html